hazelcast/src/test/java/com/hazelcast/test/TestHazelcastInstanceFactory.java

```diff
@@ -149,10 +151,10 @@ public void terminateAll() {
     private static List<Address> createAddresses(AtomicInteger ports, String... addressArray) {
         checkElementsNotNull(addressArray);
 
-        final int count = addressArray.length;
+        int count = addressArray.length;
         List<Address> addresses = new ArrayList<Address>(count);
-        for (int i = 0; i < count; i++) {
-            addresses.add(createAddress(addressArray[i], ports.incrementAndGet()));
+        for (String anAddressArray : addressArray) {
```

Comment:
anAddressArray to indicate a field of type string? I think it should be 'address'.